apps/charterafrica/src/components/Documents/Documents.js

```diff
@@ -85,6 +87,7 @@ const Documents = React.forwardRef(function Documents(props, ref) {
       setDocuments(foundDocuments);
       setPinnedDocuments(newPinnedDocuments);
       setPage(currentPage);
+      setDocuments(foundDocuments);
```

Comment:
This is already called here [here](https://github.com/CodeForAfrica/ui/blob/2c8a45bad3b859b81e7da1ea32c56788b3929061/apps/charterafrica/src/components/Documents/Documents.js#L87)